### PR TITLE
Update `d3` to address Dependabot issues

### PR DESCRIPTION
**Pull request by Etheryte**
## What does this PR change?

See title.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: version bump.

- [x] **DONE**

## Links

Part of https://github.com/SUSE/spacewalk/issues/22581

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: web/spacewalk-web.spec [hunk 1]
```diff
@@ -67,7 +67,7 @@ but it does generate a number of sub-packages.
 
 %package -n spacewalk-html
 Summary:        HTML document files for Spacewalk
-License:        (MPL-2.0 OR Apache-2.0) AND 0BSD AND BSD-3-Clause AND GPL-2.0-only AND ISC AND LGPL-3.0-or-later AND MIT AND MPL-2.0
+License:        (MPL-2.0 OR Apache-2.0) AND 0BSD AND BSD-3-Clause AND GPL-2.0-only AND ISC AND LGPL-3.0-or-later AND MIT AND MPL-2.0 AND Unlicense
```
[the remark this answers is not in the document]
Comment (Etheryte) [reply web/spacewalk-web.spec:70]: Not sure if I see what you mean? This license list is generated based on what we depend on, it isn't handwritten. [Unlicense](https://en.wikipedia.org/wiki/Unlicense) is a software license.

[the remark this answers is not in the document]
Comment (Etheryte) [reply web/spacewalk-web.spec:70]: Ping @cbbayburt.

**Review by parlt91 — APPROVED** [review]
Looks good

**Comment by Etheryte** [comment]
Closing this as we're gonna drop the whole virtualization page and this will no longer be relevant.

**Pull request closed without merging**